dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -121,10 +120,11 @@ janino/3.0.16//janino-3.0.16.jar
 javassist/3.25.0-GA//javassist-3.25.0-GA.jar
 javax.jdo/3.2.0-m3//javax.jdo-3.2.0-m3.jar
 javolution/5.5.1//javolution-5.5.1.jar
+jaxb-api/2.2.11//jaxb-api-2.2.11.jar
```

Comment:
this is CDDL 1.0 - see below for license tasks

Reply:
Updated.